tests/karma/search/components/TestSearchPage.js

```diff
@@ -7,14 +7,14 @@ import Paginate from 'core/components/Paginate';
 import { findAllByTag, findByTag, shallowRender } from '../../../utils';
 
 describe('<SearchPage />', () => {
-  let state;
+  let props;
 
-  function render(updateState = {}) {
-    return shallowRender(<SearchPage {...Object.assign({}, state, updateState)} />);
```

Comment:
I think I didn't do that before because `<SearchPage {...{...props, ...extra}} />` looks a little weird but it's nicer than writing `Object.assign` all over.